aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -261,6 +267,71 @@ public static CreateDbInstanceRequest createDbInstanceRequest(
                 .build();
     }
 
+    static Instant stringToInstant(String restoreTimeString) {
+        if (com.amazonaws.util.StringUtils.hasValue(restoreTimeString)) {
+            try {
+                return LocalDateTime.parse(
+                                restoreTimeString,
+                                DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
+                        )
+                        .atOffset(ZoneOffset.ofHours(0))
+                        .toInstant();
+            } catch (Exception e) {
+                throw new CfnInvalidRequestException(new Exception("Unable to parse RestoreTime"));
+            }
+        }
+        return null;
+    }
+
+    static RestoreDbInstanceToPointInTimeRequest restoreDbInstanceToPointInTimeRequest(
+            final ResourceModel model,
+            final Tagging.TagSet tagSet
+    ) {
+
+        final RestoreDbInstanceToPointInTimeRequest.Builder builder = RestoreDbInstanceToPointInTimeRequest.builder()
+                .autoMinorVersionUpgrade(model.getAutoMinorVersionUpgrade())
+                .availabilityZone(model.getAvailabilityZone())
+                .copyTagsToSnapshot(model.getCopyTagsToSnapshot())
+                .customIamInstanceProfile(model.getCustomIAMInstanceProfile())
+                .dbInstanceClass(model.getDBInstanceClass())
+                .dbName(model.getDBName())
+                .dbParameterGroupName(model.getDBParameterGroupName())
+                .dbSubnetGroupName(model.getDBSubnetGroupName())
+                .deletionProtection(model.getDeletionProtection())
+                .domain(model.getDomain())
+                .domainIAMRoleName(model.getDomainIAMRoleName())
+                .enableCloudwatchLogsExports(model.getEnableCloudwatchLogsExports())
+                .enableIAMDatabaseAuthentication(model.getEnableIAMDatabaseAuthentication())
+                .engine(model.getEngine())
+                .iops(model.getIops())
+                .licenseModel(model.getLicenseModel())
+                .maxAllocatedStorage(model.getMaxAllocatedStorage())
+                .multiAZ(model.getMultiAZ())
+                .networkType(model.getNetworkType())
+                .optionGroupName(model.getOptionGroupName())
+                .port(translatePortToSdk(model.getPort()))
+                .processorFeatures(translateProcessorFeaturesToSdk(model.getProcessorFeatures()))
+                .publiclyAccessible(model.getPubliclyAccessible())
+                .restoreTime(stringToInstant(model.getRestoreTime()))
+                .sourceDBInstanceIdentifier(model.getSourceDBInstanceIdentifier())
+                .sourceDbiResourceId(model.getDbiResourceId())
```

Comment:
Just as a note:  this identifier is exclusive with `sourceDBInstanceIdentifier`

Reply:
I don't understand the phrasing of this statement